plugins/rtmp-services/data/services.json

```diff
@@ -1048,6 +1048,22 @@
                     "url": "rtmp://ingest-as-central.a.switchboard.zone/live"
                 }
             ]
+        },
+        {
+            "name": "Looch",
+            "common": false,
```

Comment:
I'm pretty sure you don't _have_ to specify `"common": false`.  False is assumed if the common property is not provided.